src/component/FileManager/MusicPlayer.js

```diff
@@ -0,0 +1,396 @@
+            //index=index;
+        }else if(this.state.looptype==2){//random
+            if(this.state.items.length<=2){
+                index=index+1;
+                if (index >= this.state.items.length) {
+                    index =  0;
+                }
+            }else{
+                while(index==this.state.currentIndex){
+                    index=Math.floor(Math.random()*this.state.items.length);
+                }
+            }
+        }
+        if(this.state.currentIndex==index){
+            this.myAudioRef.current.currentTime=0;
+            this.play();
+        }
+        this.setState({
+            currentIndex: index,
+        });
+    }
+
+    prev = () => {
+        let index = this.state.currentIndex - 1
+        if (index < 0) {
+            index = this.state.items.length - 1
+        }
+        this.setState({
+            currentIndex: index,
+        });
+    };
+    
+    next = () => {
+        let index = this.state.currentIndex + 1
+        if (index >= this.state.items.length) {
+            index =  0;
+        }
+        this.setState({
+            currentIndex: index,
+        });
+    };
+
+    handleProgress = (e, newValue) => {
+        this.myAudioRef.current.currentTime=newValue;
+    }
+      
+
+    render() {
+        const { currentIndex, items } = this.state;
+        const { isOpen,classes } = this.props;
+        return (            
+            <Dialog
+                open={isOpen}
+                onClose={this.backgroundPlay}
+                aria-labelledby="form-dialog-title"
+                maxWidth="xs"
+                fullWidth
+                keepMounted
+            >
+                <DialogTitle id="form-dialog-title">音频播放</DialogTitle>
+                <DialogContent>
+                    <List className={classes.list} dense>
+                        {items.map((value,idx) => {
+                            const labelId = `label-${value.intro}`;
+                            return (
+                                <ListItem key={value.src} dense button onClick={this.handleItemClick(idx)}
+                                    selected={idx==currentIndex}>
+                                    <ListItemIcon>
+                                        {
+                                            idx==currentIndex?(<PlayArrow/>):(<MusicNote/>)
+                                        }
+                                    </ListItemIcon>
+                                    <ListItemText id={labelId} primary={`${value.intro}`} />
+                                </ListItem>
+                            );
+                        })}
+                    </List>
+                    <audio ref={this.myAudioRef}                                 
+                        src={items[currentIndex]?.src}
+                        />
+                    <div style={{"padding-top":8}}></div>
+                    <Grid container spacing={2}>
+                        <Grid item xs>
```

Comment:
Add a new style `align-items: center;` for this `Grid` to make the audio control vertical-align.

Reply:
OK.

Reply:
Done. 
The slider is not vertically aligned to middle by default. A style patch to the slider is made.